content/general/concepts/white-spaces/white-spaces.md

```diff
@@ -0,0 +1,63 @@
+---
+Title: 'Lists'
+Description: 'White spaces are spaces and blank lines in programing'
+Subjects:
+  - 'Computer Science'
+  - 'Data Science'
+  - 'Web Development'
+Tags:
+  - 'Formatting'
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/computer-science'
+---
+
+White spaces are spaces (` `) or blanck lines. Whitespaces are used to format code to improve the readablity of code. 
+
+## Why they are important
+
+Take this as an example.
```

Comment:
```suggestion
The below code is hard to read since it has no gaps and it also does not have indention with spaces so people can not easily see what is nested within what.
```